actions/crib-deploy-environment/action.yml

```diff
@@ -128,8 +142,42 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Generate CRIB Namespace Name, configure variables
+      id: generate-ns-name
+      shell: bash
+      env:
+        NS_NAME_PREFIX: ${{ inputs.ns-name-prefix }}
+        PR_NUMBER: ${{ github.event.number }}
+        WORKFLOW_JOB: ${{ github.job }}
+        REPO_NAME: $(basename $GITHUB_REPOSITORY)
```

Comment:
Is it valid to have shell functions here? If so, TIL.

Reply:
It was working for some reason even it's not supported 🤦‍♂️ Thanks.